llama-index-core/llama_index/core/chat_engine/types.py

```diff
@@ -60,6 +61,31 @@ def __post_init__(self) -> None:
     def __str__(self) -> str:
         return self.response
 
+    @property
+    def response_gen(self) -> Generator[str, None, None]:
```

Comment:
Yes -- if return_direct happens while streaming, we create a dummy stream (since the user will still be expecting to stream something, but we can't create a streaming response)

Reply:
Hmm. To me, `null_stream` means there is no stream? 🤔 There is one, it's just being faked. `enable_fake_stream` maybe?

Reply:
Ah okay. Good point. I think dummy is good then!